sshd-core/src/main/java/org/apache/sshd/common/session/helpers/AbstractSession.java

```diff
@@ -254,6 +253,46 @@ protected AbstractSession(
         }
     }
 
+    /**
+     * @param  len       The packet payload size
+     * @param  blockSize The cipher block size
+     * @param  etmMode   Whether using &quot;encrypt-then-MAC&quot; mode
+     * @return           The required padding length
+     */
+    protected static int calculatePadLength(int len, int blockSize, boolean etmMode) {
```

Comment:
Let's make it public - even though used only by sessions - perhaps someone else (not our code) uses it.

Reply:
Done